HIPs/incubation/web3j.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: default
+title: Web3j
+parent: Hyperledger Improvement Proposals
+---
+
+# HIP Identifier
+
+Hyperledger Web3j v1.0
+
+# Sponsor(s)
```

Comment:
Based on the [Incubation Entry Considerations](https://toc.hyperledger.org/guidelines/project-incubation-entry-considerations.html), 
> Sponsors are advocates for the project. There should be more than one sponsor, and they should be from different organizations. They may or may not be committing resources to the project.

Do we have any other organizations or individuals that would be willing to be sponsors?

Reply:
Not yet, but I haven't started reaching out to people.

There are some member organisations in Hyperledger that are potentials, but it would also be useful if there was a mechanism to do this to put out a general expression of interest or equivalent among members.